core/js/src/main/scala/cats/effect/unsafe/FiberAwareExecutionContext.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020-2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+package unsafe
+
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext
+
+private[effect] final class FiberAwareExecutionContext(ec: ExecutionContext)
+    extends ExecutionContext {
+
+  def fibers: Set[IOFiber[_]] = fiberBag.toSet
```

Comment:
```suggestion
  def liveFibers(): Set[IOFiber[_]] = fiberBag.toSet
```

The `()` will match the JVM code, which will be unsafe and contain side effects.

Reply:
👍 I think we may eventually need this behind a shared trait, so that code can do `isInstanceOf[FiberAwareEC]` or something to check if this method is available.

Reply:
I have a plan, don't you worry ;)